src/index.tsx

```diff
@@ -549,6 +585,8 @@ class App extends React.Component<{}, AppState> {
       });
       this.forceUpdate();
       event.preventDefault();
+    } else if (shapesShortcutKeys.includes(event.key.toLowerCase())) {
```

Comment:
This `.toLowerCase()` ensures that if the user has forgotten caps-lock on, the shortcut will be triggered

Reply:
Good catch, we'll need to update the other shortcuts to handle this properly!

Reply:
Do you want me to update the `Cmd + a` case too?

Reply:
If you can yeah, would be good